### PR TITLE
Feat/uploadthing-file-upload

### PR DESCRIPTION
**Pull request by CedricAngeloSina-on**

### CODE REVIEW

File: src/app/api/uploadthing/core.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+            // If you throw, the user will not be able to upload
+            if (!user) throw new UploadThingError("Unauthorized");
+
+            // Whatever is returned here is accessible in onUploadComplete as `metadata`
+            return { userId: user.id };
+        })
+        .onUploadComplete(async ({ metadata, file }) => {
+            // This code RUNS ON YOUR SERVER after upload
+            console.log("Upload complete for userId:", metadata.userId);
```
Comment (evad-e) [inline src/app/api/uploadthing/core.ts:27]: Remove console statements

File: src/components/uploadthing-upload-button.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+"use client";
+import { UploadButton } from "~/utils/uploadthing";
+
+export default function HomePage() {
+    return (
+        <UploadButton
+            endpoint="csvUploader"
+            onClientUploadComplete={(res) => {
+                // Do something with the response
```
Comment (evad-e) [inline src/components/uploadthing-upload-button.tsx:9]: Cleanup unnecessary comments

**Review by evad-e — CHANGES_REQUESTED** [review]

File: src/components/uploadthing-upload-button.tsx (hunk 2 above)
Comment (CedricAngeloSina-on) [reply src/components/uploadthing-upload-button.tsx]: removed comments

File: src/app/api/uploadthing/core.ts (hunk 1 above)
Comment (CedricAngeloSina-on) [reply src/app/api/uploadthing/core.ts]: removed console logs but added TODO comment for file.url to send to flask API

**Review by evad-e — APPROVED** [review]

**Pull request merged**